companies/benjerry.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Ben & Jerry's
+slug: ben-&-jerrys
```

Comment:
```suggestion
slug: ben-jerrys
```

The slug value is used for a url, so I think we might have problems if we leave the `&` in there.  Let's be safe and remove that special character.

Reply:
If you want to learn more, [this article](https://searchengineland.com/seo-friendly-url-syntax-practices-134218) has a section on Reserved Characters ⭐